UPGRADING.rst

```diff
@@ -10,6 +10,22 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#4011 Integrate monitoring SNS topic with Slack
+===============================================
+
+Operator
+~~~~~~~~
+
+Complete the following steps before pushing the merge commit to the GitLab
+instance in any main deployment. Set up AWS Chatbot Slack client as `described
+<./README.md#313-aws-chatbot-integration-with-slack>`_ in the README. This is
+done for the ``platform-hca-dev`` account but is still needed for
+``platform-anvil-dev`` and ``platform-hca-prod``.
+
+Once Chatbot is set up, manually deploy the ``shared`` component in that
+deployment.
```

Comment:
It's ambiguous if this is already done or still needed for platform-hca-dev.

Reply:
I removed the first sentence. And there's also the PR checklist item "Follow upgrading instructions for `dev.shared`" which I hope rids of ambiguity.